docs/sources/clients/promtail/stages/drop.md

```diff
@@ -0,0 +1,179 @@
+---
+title: drop
+---
+# `drop` stage
+
+The `drop` stage is a filtering stage that lets you drop logs based on several options. 
+
+It's important to note that if you provide multiple options they will be treated like an AND clause,
+where each option has to be true to drop the log.
+
+If you wish to drop with an OR clause, specify multiple drop stages.
+
+There are examples below to help explain. 
+
+## Schema
```

Comment:
Which scheme? The drop scheme? Please try to be specific. Pretend you're trying to find this with a web search.